action-src/main.ts

```diff
@@ -62,6 +62,13 @@ const getBuildInfo = (event: typeof context) => {
         slug: event.payload.repository.full_name,
       };
     }
+    case 'release': {
+      return {
+        sha: event.sha,
+        branch: event.ref.replace('refs/tags/', ''),
```

Comment:
Maybe we should use the tag as the `branch` name instead of the target commitish?

```suggestion
        branch: event.payload.release.target_commitish('refs/tags/', ''),
```

Reply:
Good